src/components/login/login.tsx

```diff
@@ -32,7 +34,13 @@ async function login(
 
     return "";
   } catch (error) {
-    return error as string;
+    let message = "Unknown Error";
+    if (error instanceof Error) {
+      message = error.message;
+    } else {
+      message = String(error);
+    }
+    return message;
```

Comment:
the caller is expecting a string but `error` is a `Error()` object. extract the message or try and cast to a string otherwise.